js/script.js

```diff
@@ -0,0 +1,41 @@
+import { $taskInput, $taskContainer, $addButton, $errorMessage} from "./element.js";
+import { sanitizeInput } from "./utility.js";
+
+let taskList = [];
```

Comment:
`users`, `tasks` is more concise than `userList` and `taskList`
```suggestion
let tasks = [];
```